Infrastructure/Data/SeedData/SeedData.cs

```diff
@@ -0,0 +1,936 @@
+﻿using Bogus;
+using Core.Entities.CarCategoryEntity;
+using Core.Entities.CarEntity;
+using Core.Entities.GoodCategoryEntity;
+using Core.Entities.OfferEntity;
+using Core.Entities.PointEntity;
+using Core.Entities.RoleEntity;
+using Core.Entities.UserEntity;
+using Core.Roles;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.EntityFrameworkCore;
+using System;
+
+namespace Infrastructure.Data.SeedData
+{
+
+    public static class SeedData
+    {
+        #region Users
+
+        private static readonly string VADIM_ID = Guid.NewGuid().ToString();
+        private static readonly string VLAD_ID = Guid.NewGuid().ToString();
+        private static readonly string ANDREW_ID = Guid.NewGuid().ToString();
+        private static readonly string MARYNA_ID = Guid.NewGuid().ToString();
+        private static readonly string EUGEN_ID = Guid.NewGuid().ToString();
+        private static readonly string VOLODYA_ID = Guid.NewGuid().ToString();
+        private static readonly string ANTONINA_ID = Guid.NewGuid().ToString();
+        private static readonly string SERGEY_ID = Guid.NewGuid().ToString();
+        private static readonly string USER_ROLE_ID = Guid.NewGuid().ToString();
+
+        #endregion
+
+        private static readonly string[] Colors = new[] 
+            {"Червоний","Чорний","Білий","Сірий","Зелений","Синій","Жовтий"};
+        private static readonly PasswordHasher<User> PasswordHasher = new PasswordHasher<User>();
+        private static readonly Faker Faker = new Faker();
+        private static float _maxWeight = 1000f;
+
+        public static void Seed(this ModelBuilder builder)
```

Comment:
Why does the owner need so much data, including personal data of developers?
I think you should have created only roles, categories and one user (with random data, not real) by default